WordPress/Classes/Models/Post.h

```diff
@@ -10,26 +12,26 @@ extern NSString * const PostTypeDefaultIdentifier;
 ///-------------------------------
 /// @name Specific Post properties
 ///-------------------------------
-@property (nonatomic, strong) NSNumber *commentCount;
-@property (nonatomic, strong) NSNumber *likeCount;
-@property (nonatomic, strong) Coordinate *geolocation;
-@property (nonatomic, strong) NSString *tags;
-@property (nonatomic, strong) NSString *postType;
-@property (nonatomic, strong) NSString *postFormat;
-@property (nonatomic, strong) NSString *postFormatText;
-@property (nonatomic, strong) NSSet *categories;
+@property (nonatomic, strong, nullable) NSNumber *commentCount;
+@property (nonatomic, strong, nullable) NSNumber *likeCount;
+@property (nonatomic, strong, nullable) Coordinate *geolocation;
+@property (nonatomic, strong, nullable) NSString *tags;
+@property (nonatomic, strong, nullable) NSString *postType;
```

Comment:
I've been thinking about this extensively, because I'm not sure the nullability info in CoreData can match the nullability info in our code in all cases.

Here's the reason: nullability in CoreData is only verified when saving changes (to the context), while nullability in Swift refers to the entire lifecycle of an object.

With Objective-C code, we won't see a problem setting it either way, because its runtime is very flexible and allows `nonnull` objects to be `nil`.

However if we migrated `Post` to Swift (which I'm tackling currently), I believe we would find out this property would need to be optional, since it doesn't have any initializer setting an initial value.

So long story short: would you mind me leaving this as optional until I can finish the migration to Swift?  It shouldn't be long and I'll make sure to /cc you on that PR.  If I can make this property non-optional I'll make sure to do it then.